src/components/Registration/RegistrationStepTwo/RegistrationStepTwo.tsx

```diff
@@ -113,7 +161,11 @@ export const RegistrationStepTwo = () => {
                         </Button>
                       </Col>
                       <Col size={6} className="text-right">
-                        <Button color="primary" className="w-50">
+                        <Button
+                          color="primary"
+                          className="w-50"
+                          onClick={() => console.log(props.selectedInstitution)}
```

Comment:
is this on purpose ?

Reply:
yes, the service to save an institution is not ready yet so the correct action to link to this button's onClick can't be added
I can leave the button without any action if you prefer

Reply:
lgtm, i suggest to leave a comment for the reviewers to mark dangling code as on purpose. the ideal is to put a reference to the pivotal task that will complete it, ie: 

// TODO: add X and Y see https://www.pivotaltracker.com/story/show/12345678